### DISCUSSION TITLE
why can not find plot?

### DISCUSSION THREAD

**Discussion by gaodaqiang1** [Chia-Network/chia-blockchain#18935] [Support]
![image](https://github.com/user-attachments/assets/cea2ce91-b2c6-46f3-8887-b92f28d137b4)

**Comment by BrandtH22** [comment]
Hey @gaodaqiang1 , as a heads up we can generally provide more timely and thorough support in our discord server (https://discord.gg/chia).

I recommend running a chia plots check on the drive to make sure the chia software can pick them up properly: https://docs.chia.net/cli/#plots-check

`chia plots check -g D:\plot`

Let us know the results of that command or if any issues arise while running it in powershell.

**Comment by BrandtH22** [comment]
No response from OP, please reopen the ticket if this issue persists
